docs/pjrt_integration.md

```diff
@@ -28,18 +42,21 @@ PJRT_Error* PJRT_Client_Create(PJRT_Client_Create_Args* args) {
 }
 }  // namespace my_plugin
 ```
-  Note [PJRT\_Client\_Create](https://github.com/openxla/xla/blob/c23fbd601a017be25726fd6d624b22daa6a8a4e5/xla/pjrt/c/pjrt_c_api.h#L344-L365) can take options passed from the framework. [Here](https://github.com/openxla/xla/blob/c23fbd601a017be25726fd6d624b22daa6a8a4e5/xla/pjrt/c/pjrt_c_api_gpu_internal.cc#L48-L102) is an example of how a GPU client uses this feature.
+
+  Note [PJRT\_Client\_Create](https://github.com/openxla/xla/blob/b8ec2c4c9dcccaf33b548bee2c4c33a778a8cb88/xla/pjrt/plugin/example_plugin/myplugin_c_pjrt.cc#L33-L38) can take options passed from the framework. [Here is an example of how a GPU client uses this feature](https://github.com/openxla/xla/blob/c23fbd601a017be25726fd6d624b22daa6a8a4e5/xla/pjrt/c/pjrt_c_api_gpu_internal.cc#L43-L115).
+
+  Also note that `PJRT_Client_Create` [needs extra arguments](https://github.com/openxla/xla/blob/a7d1ed8a9091bf51aed427659218548559152be2/xla/pjrt/c/pjrt_c_api_wrapper_impl.h#L459) (they can all be unimplemented methods).
```

Comment:
Please let me know if this wording is correct.

Reply:
cc @changm